### PR TITLE
Update todo list items

### PR DESCRIPTION
**Pull request by divisandeep94**
This PR should be merged after this PR - https://github.com/divisandeep94/todo-app-poc/pull/2

### CODE REVIEW

File: app/components/create-todo/index.test.tsx [hunk 1]
```diff
@@ -1,58 +1,87 @@
...
+    const inputEl = screen.getByPlaceholderText(appLabels.NEW_TODO_INPUT_PLACEHOLDER_TEXT)
+    expect(inputEl.textContent).toEqual('')
+    const testValue = faker.word.words()
+    fireEvent.change(inputEl, { target: { value: testValue } })
+    expect(inputEl).toHaveValue(testValue)
+  })
+
+  it('onclick of cancel should re-direct to home page', async () => {
+    const push = jest.fn()
+    ;(useRouter as jest.Mock).mockImplementation(() => ({
```
Comment (kool7) [inline app/components/create-todo/index.test.tsx]: this seems a typo.

File: app/test-utils/test-constants.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export const TestConstants = {}
```
Comment (kool7) [inline app/test-utils/test-constants.ts]: why empty object?

File: app/components/create-todo/index.test.tsx [hunk 3]
```diff
@@ -1,58 +1,87 @@
...
+  it('onclick of create should create new todo item', () => {
+    const push = jest.fn()
+    ;(useRouter as jest.Mock).mockImplementation(() => ({
+      push,
+    }))
+    render(
+      <TestProvider initialValues={[[todoListAtom, todoItems]]}>
+        <ListItem />
+        <CreateTodo />
+      </TestProvider>
```
Comment (aashikvilla) [inline app/components/create-todo/index.test.tsx:78]: These kind of interaction tests (like add/edit/delete and verifying on list) can part of page.tsx tests, there we would not need to add the provider and components explicitly. 
I'm fine this way as well.

File: app/components/list-item/index.test.tsx [hunk 4]
```diff
@@ -97,4 +59,59 @@ describe('List-item Component', () => {
     expect(newEditIconElements.length).toEqual(listElements.length - 2)
     expect(newBinIconElements.length).toEqual(listElements.length - 2)
   })
+
+  it('should delete that todo item, when delete icon is clicked', () => {
+    // Adding this manual mock data to check the right label value gets deleted or not.
```
Comment (aashikvilla) [inline app/components/list-item/index.test.tsx]: We should be able to use generateRandomData() just like other places.

**Review by aashikvilla — APPROVED** [review]
Good work!

File: app/components/create-todo/index.test.tsx (hunk 3 above)
Comment (divisandeep94) [reply app/components/create-todo/index.test.tsx:78]: I agree with you Aashik. I'll plan the test cases that way going forward. 

Thank you for your feedback!

**Review by kool7 — COMMENTED** [review]
looks good.

**Comment by divisandeep94** [comment]
This PR should be merged after this PR - https://github.com/divisandeep94/todo-app-poc/pull/3

**Pull request merged**